spec/defines/server/instances_spec.rb

```diff
@@ -3,131 +3,154 @@
 require 'spec_helper'
 
 describe 'ssh::server::instances' do
-  context 'with sftp_server present' do
```

Comment:
if you like, I can pull the refactoring of this file into a dedicated PR so we can review it separatly.